parcels/field.py

```diff
@@ -190,6 +190,11 @@ def __init__(self, name, data, lon=None, lat=None, depth=None, time=None, grid=N
         # self.lon, self.lat, self.depth and self.time are not used anymore in parcels.
         # self.grid should be used instead.
         # Those variables are still defined for backwards compatibility with users codes.
+        if np.all(self.grid.lat[:-1] > self.grid.lat[1:]):
```

Comment:
Like every new feature in Parcels, this will also need a unit test. Probably in `test_fieldset.py`, it would be good to create a small test function that creates a latitude vector in 'reverse' order, then loads it using `fieldset.from_data()` and then checks if the latitude (and data array) are indeed correctly reversed